### ISSUE TITLE
EXTRA_VALIDATE_EXPIRY_DATE variable not present in CreditCardUtils

### ISSUE THREAD

**Issue by Shajeel-Afzal** [cooltechworks/CreditCardView#11]
I have copied the code from https://github.com/cooltechworks/CreditCardView#edit-existing-card for editing the card information.

The code gives error at `intent.putExtra(CreditCardUtils.EXTRA_VALIDATE_EXPIRY_DATE, true);` because it is not able to find the `CreditCardUtils.EXTRA_VALIDATE_EXPIRY_DATE`. I have checked the `CreditCardUtils`. 

Please solve or update the readme file.

**Comment by cooltechworks** [comment]
Oops. Will do that.

Thanks for notifying.

-Harish

**Comment by cooltechworks** [comment]
Resolved in d03b24a. Use new version as gradle dependency.

```gradle
dependencies {
 compile 'com.github.cooltechworks:CreditCardView:v1.0.1'
}
```
